src/getting-started.rst

```diff
@@ -48,6 +52,9 @@ If you want more advanced ways to run ActivityWatch (including running it withou
 .. note::
    If you are running GNOME 3 or another desktop environment that does not support system trays, or if for some reason Qt can't be used on your machine, read `Running on GNOME`.
 
+.. note::
+   If your Linux system is using Wayland rather than X11, the default watchers will not work. Read :ref:`window and idle watchers for Wayland<wayland-watchers>`.
```

Comment:
```suggestion
   If your Linux system is using Wayland rather than X11, the default watchers will not work. See :ref:`window watchers <window-watchers>`.
```

Reply:
Thanks!